cloud-consumption/src/main/java/com/sequenceiq/consumption/flow/consumption/storage/event/SendStorageConsumptionEvent.java

```diff
@@ -14,8 +17,14 @@ public SendStorageConsumptionEvent(String selector, Long resourceId, String reso
         this.storageConsumptionResult = storageConsumptionResult;
```

Comment:
shouldn't the same be added to this constructor as well?

Reply:
I think Jackson allow only one constructor to be creator that's why.

Reply:
True, only one `@JsonCreator` is allowed
(ok, it can be hacked in a way, but let's just not consider that here :) )